### PR TITLE
Limit visibility of Storage, Weight, and Locations panels for ppm tab

### PR DESCRIPTION
**Pull request by ralren**
## Description
If a service member hasn't requested payment yet, an office user should not be able to see Storage, Weight, and Locations panels.

## Reviewer Notes

Please take a look at the e2e tests and the changes made in `e2ebasic.go` as I'm not confident the path taken to satisfy the tests was the best one taken.

## Setup

```sh
make server_run
make office_client_run
```
1. Log in as an office user.
2. Find the move with the locator code "PAYMNT"
3. Go to the PPM Panel.
4. Notice all the panels are there.
5. Compare with another move whose status isn't "Payment Requested" or "Completed"
6. Notice the Storage, Weight, and Locations panels are not present.

### Tests
```sh
make e2e_test
```
Run the `officeUserPPM.js` and `landingPages.js` tests.

## Code Review Verification Steps
* [x] User facing changes have been reviewed by design.
* [x] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/165003101) for this change

## Screenshots
Screen Shot 2019-04-11 at 9 06 20 AM

### CODE REVIEW

File: pkg/testdatagen/scenario/e2ebasic.go [hunk 1]
```diff
@@ -365,6 +373,52 @@ func (e e2eBasicScenario) Run(db *pop.Connection, loader *uploader.Uploader, log
 	ppm2.Move.PersonallyProcuredMoves[0].Approve(time.Now())
 	models.SaveMoveDependencies(db, &ppm2.Move)
 
+	/*
+	 * Service member with a ppm move with payment requested
```
Comment (kahlouie) [inline pkg/testdatagen/scenario/e2ebasic.go]: Next time you add a new move, I'd suggest adding it to the end and leave existing data so you don't have to increment all of the same move types following. I usually copy the one closest to what I want, then edit all the uuids and variable names as needed. :)

File: src/scenes/Landing/StatusTimeline.jsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ export class PPMStatusTimeline extends React.Component {
 
     if (status === 'IN_PROGRESS') {
       const moveInProgress = moment(ppm.original_move_date, 'YYYY-MM-DD').isSameOrBefore();
-      return moveInProgress && includes(['APPROVED', 'PAYMENT_REQUESTED', 'COMPLETED'], ppm.status);
+      return moveInProgress || includes(['APPROVED', 'PAYMENT_REQUESTED', 'COMPLETED'], ppm.status);
```
Comment (kahlouie) [inline src/scenes/Landing/StatusTimeline.jsx]: Are we sure that this is accurate? Can a move be in progress if it hasn't been at minimum approved?

File: pkg/testdatagen/scenario/e2ebasic.go (hunk 1 above)
Comment (kahlouie) [reply pkg/testdatagen/scenario/e2ebasic.go]: The reason this is important is that ppm4 already exists lower down in the file: https://github.com/transcom/mymove/blob/2995562b057186a8d8efc2f0e350ac994879711a/pkg/testdatagen/scenario/e2ebasic.go#L2643, as does at least a ppm5.

File: src/scenes/Landing/StatusTimeline.jsx (hunk 2 above)
Comment (ralren) [reply src/scenes/Landing/StatusTimeline.jsx]: Good question. I honestly don't know. The reason I made this change is that when the original move date is in the future and the status is already in `PAYMENT_REQUESTED` or `COMPLETED`, the timeline is messed up with the in-progress being failed to mark as completed while the phases ahead of it are.

File: pkg/testdatagen/scenario/e2ebasic.go (hunk 1 above)
Comment (ralren) [reply pkg/testdatagen/scenario/e2ebasic.go]: Ah, yes, I will do that.

File: cypress/integration/office/officeUserPPM.js [hunk 3]
```diff
@@ -33,6 +33,13 @@ describe('office user finds the move', function() {
       .contains('Approved')
       .should('be.disabled');
   });
+
+  it('office user can not see storage, weight, and locations panels', function() {
```
Comment (mkrump) [inline cypress/integration/office/officeUserPPM.js]: maybe add just a bit more about about why they shouldn't see the moves. something along the lines of 
"for moves for which the service member hasn't requested payment, ...."

**Pull request merged**